### PR TITLE
feat: Add EF repositories

### PR DESCRIPTION
**Pull request by taustad**
[AB#121023](https://dev.azure.com/EquinorASA/bb9bd8cb-74f7-4ffa-b0cb-60eff0a0be58/_workitems/edit/121023)

### CODE REVIEW

File: datasheetapi/Repositories/CommentRepository.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    public async Task DeleteComment(Comment comment)
+    {
+        _context.Comments.Remove(comment);
+        await _context.SaveChangesAsync();
+    }
+
+    public async Task<Comment> UpdateComment(Comment comment)
+    {
+        var existingComment = await _context.Comments.FindAsync(comment.Id) ?? throw new ArgumentException("Comment not found");
+        existingComment.Text = comment.Text;
```
Comment (fredrikleknes) [inline datasheetapi/Repositories/CommentRepository.cs:39]: Wasn't the logic here supposed to be in the service?

File: tests/Adapters/TagDataReviewAdapterTests.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public void ToDtoOrNull_WithNonNullTagDataReview_ReturnsTagDataReviewDto()
             Id = Guid.NewGuid(),
             CreatedDate = DateTime.UtcNow,
             ModifiedDate = DateTime.UtcNow,
-            TagDataId = Guid.NewGuid(),
+            TagNo = "Tag-001",
```
Comment (fredrikleknes) [inline tests/Adapters/TagDataReviewAdapterTests.cs:31]: duplicating this line is causing the linting error fyi

File: datasheetapi/Repositories/CommentRepository.cs (hunk 1 above)
Comment (taustad) [reply datasheetapi/Repositories/CommentRepository.cs]: Moved the logic to the service

**Review by fredrikleknes — APPROVED** [review]
tested locally lgtm

**Pull request merged**